web/api/views.py

```
@@ -1150,7 +1145,7 @@ def abort_subscan(subscan):
 			try:
 				scan = ScanHistory.objects.get(id=scan_id)
 				# if scan is already successful or aborted then do nothing
-				if scan.scan_status in (SUCCESS_TASK, ABORTED_TASK):
+				if scan.scan_status == SUCCESS_TASK or scan.scan_status == ABORTED_TASK:
```

Ruff

`SUCCESS_TASK` may be undefined, or defined from star imports

(F405)